ui/selectors/selectors.js

```diff
@@ -2982,7 +2984,23 @@ export function getMetaMetricsDataDeletionStatus(state) {
   return state.metamask.metaMetricsDataDeletionStatus;
 }
 
+/**
+ * Gets the remote feature flags from either the manifest or state.
+ * First checks if remote feature flags exist in the manifest and returns those if present.
+ * Otherwise returns the remote feature flags from the MetaMask state that's retrieved from controller.
+ *
+ * @param {object} state - The MetaMask state object
+ * @returns {object} The remote feature flags object containing feature flag key-value pairs
+ */
 export function getRemoteFeatureFlags(state) {
```

Comment:
5. In UI selector, `remoteFeatureFlagsValueFromManifest` gets priority to be returned, then we will read from controller.